curvefs/src/mds/fs_storage.cpp

```diff
@@ -186,14 +188,21 @@ FSStatusCode MemoryFsStorage::DeleteFsUsage(const std::string& fsName) {
     return FSStatusCode::OK;
 }
 
+FSStatusCode MemoryFsStorage::Tso(uint64_t* ts, uint64_t* timestamp) {
+    *timestamp = curve::common::TimeUtility::GetTimeofDayMs();
+    *ts = tsId_.fetch_add(1, std::memory_order_relaxed);
```

Comment:
Is all `tso` has increasing sequence number, not divide into every timestamp?

Reply:
Yes, tso has a monotonically increasing sequence number and a timestamp which used to determine tx lock timeout.